index.html

```diff
@@ -90,16 +92,20 @@ <h3 > Functional and non-Functional </h3>
 			<h5> Functional </h5>
 			<ul>
 				<li> The user should be able to get to any reference page</li>
-				<li>Send a email from the contact us page </li>
-				<li>The location can be shown with a google map </li>
-				<li>There will be dropdown menus </li>
+				<li>The user should be able to contact the buisness by mail </li>
+				<li>The location should be easily found </li>
```

Comment:
Bedre at det står med google map

Reply:
Men når det står med google maps, blir ikke det mer mot non functional?

Reply:
hmm nei for studass sa at non functional er mer hvor fort en side laster inn. Mer hvordan det bak fungerer. Hvis det gir mening.